connector/redis.go

```diff
@@ -69,3 +70,55 @@ func GetRedisConnectionWithAuth(ctx context.Context) *redis.ClusterClient {
 	})
 	return redisClusterClient
 }
+
+func SetRedisKey(key string, data string, exp time.Duration) error {
+	ctx := context.Background()
+	GetRedisConnectionWithAuth(ctx)
+	return redisClient.Set(ctx, key, data, exp).Err()
+}
+
+func SetRedisKeyObject(key string, data interface{}, exp time.Duration) error {
+	ctx := context.Background()
+	GetRedisConnectionWithAuth(ctx)
+	return redisClient.Set(ctx, key, data, exp).Err()
```

Comment:
`GetRedisConnectionWithAuth` function,  set client in `redisClusterClient` variable, Here `redisClient ` is getting used to set key. We need to use `redisClusterClient, Please change in all the places